content/blog/2022-05-24-MLEM-release.md

```diff
@@ -0,0 +1,278 @@
+training workflows by adding a couple of lines:
+
+```python
+import mlem
+mlem.api.save(
+    my_model,
+    "mlem-model",
+    sample_data=train
+)
+```
+
+That produces two files: model binary and model metadata, which is a `.mlem`
+file:
+
+```shell
+$ ls models
+mlem-model mlem-model.mlem
+```
+
+MLEM automatically detects everything you need to run the model: ML framework
+you use, as well as model dependencies (i.e. Python requirements), methods and
+input/output data schema. MLEM saves this information in a simple human-readable
+YAML file:
+
+```yaml
+artifacts:
+  data:
+    hash: b7f7e869f2b9270c516b546f09f49cf7
+    size: 166864
+    uri: mlem-model
+description: Random Forest Classifier
+labels:
+  - random-forest
+  - classifier
+model_type:
+  methods:
+    predict_proba:
+      args:
+        - name: data
+          type_:
+            columns:
+              - sepal length (cm)
+              - sepal width (cm)
+              - petal length (cm)
+              - petal width (cm)
+            dtypes:
+              - float64
+              - float64
+              - float64
+              - float64
+            index_cols: []
+            type: dataframe
+      name: predict_proba
+      returns:
+        dtype: float64
+        shape:
+          - null
+          - 3
+        type: ndarray
+  type: sklearn
+object_type: model
+requirements:
+  - module: sklearn
+    version: 1.0.2
+  - module: pandas
+    version: 1.4.1
+  - module: numpy
+    version: 1.22.3
+```
+
+# Running your models anywhere
+
+The main goal of MLEM is to provide you a single tool that enables any kind of
+model usage scenarios. Three main groups of those are:
+
+- using the model with the MLEM's help,
+- exporting a model to a format that can be used by other tools,
+- and deploying a model to a production environment.
+
+The first one allows you to import your model into a Python runtime, run predict
+against some dataset directly in the command line, or serve the model with MLEM
+from your CLI.
+
+```python
+$ python
+>>> import mlem
+>>> model = mlem.api.load("mlem-model")
+>>> model.predict(test)
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem apply mlem-model test.csv
+[[0.4, 0.3, 0.3], [0.2, 0.5, 0.3]]
+```
+
+```shell
+$ mlem serve ml-model fastapi
+⏳️ Loading model from ml-model.mlem
+Starting fastapi server...
+💅 Adding route for /predict
+💅 Adding route for /predict_proba
+Checkout openapi docs at <http://0.0.0.0:8080/docs>
+INFO:     Started server process [5750]
+INFO:     Waiting for application startup.
+INFO:     Application startup complete.
+INFO:     Uvicorn running on http://0.0.0.0:8080 (Press CTRL+C to quit)
+```
+
+The second one allows you to package your models as a Python package, build a
+Docker Image, or export it as some other format (e.g. `.onnx` which is coming
+soon).
+
+```shell
+$ mlem pack mlem-model pip -c package_name=mlem-translate -c target=build/
```

Comment:
So same question on name changing discussion that happened today.  Effects this, right?